config/mfDefsGEM.mk

```diff
@@ -155,8 +164,8 @@ else
 $(info Unable to determine flags for gcc($(GCCNUM))/clang($(CLANGNUM)))
 endif
 
-UserCFlags+=-O0 -g3 -fno-inline
-UserCCFlags =${UserCFlags}
+## we should really keep these separate, no? C vs C++...
```

Comment:
Agreed, even if some flags will be duplicates. So, why making the variables identical? ;)